eZ/Bundle/EzPublishCoreBundle/Resources/views/default/content/full.html.twig

```diff
@@ -0,0 +1,5 @@
+<h2>{{ ez_content_name(content) }}</h2>
+{% for field in content.fieldsByLanguage %}
+    <h3>{{ field.fieldDefIdentifier }}</h3>
+    {{ ez_render_field(content, field.fieldDefIdentifier) }}
+{% endfor %}
```

Comment:
IMO most functional tests should *not* depend on demo bundle as that one will change from release to release. So, having templates which are somehow functional when installing "clean" would be convenient.

Having okay base templates would help avoiding the kind of mess we ended up with regarding fallback templates in legacy ( standard -> plain -> base -> webin -> ezflow )

My two cents...

Anyway, this PR has been merged now, so I guess some other time maybe....